cmd/makedocs/main.go

```diff
@@ -59,15 +84,62 @@ func main() {
 	}
 }
 
-func getDesc(name string) (string, error) {
-	b, err := ioutil.ReadFile(checkersPath + name + ".md")
-	return string(b), err
-}
+type parseState uint
+
+const (
+	emptyLine parseState = 1 << iota
+	desciption
+	before
+	after
```

Comment:
Usually, we prefix enum values with some common prefix that resembles their type.
`stEmptyLine`, `stDescriotion`, etc.
Yeah, like in C. :)